### DISCUSSION TITLE
What's new for Developer (March 2023)

### DISCUSSION THREAD

**Discussion by bph** [WordPress/developer-blog-content#80] [Topic Ideas]
This is a rolling discussion for a new monthly series that is intended to keep track of the items, PRs, Trac ticket and make blog posts that are interesting/important for developers. The current goal is to publish a post on the 10th of each month with the previous month's updates.

Everyone is welcome to contribute to this monthly document. If you find something interesting, just create a new reply to this discussion below.

Resources to track:

- [Keeping up with Gutenberg: Index 2022](https://make.wordpress.org/core/handbook/references/keeping-up-with-gutenberg-index/)
- [A Week in Core](https://make.wordpress.org/core/tag/week-in-core/)
- [Gutenberg plugin release posts](https://make.wordpress.org/core/tag/gutenberg-new/)
- [Core editor chat](https://make.wordpress.org/core/tag/core-editor-summary/), [dev chat](https://make.wordpress.org/core/tag/dev-chat+summary/), [performance team](https://make.wordpress.org/core/tag/performance-chat+summary/) meeting summaries

@justintadlock created a new Google Doc for March: https://docs.google.com/document/d/1l2CGDiXFRN6QhEzdiykGx779ozMehUM1ZnXD66ZbKd8/edit#

**Comment by bph** [comment]
[[Twenty Twelve to Twenty Seventeen]: bundle Google Fonts in each theme](https://core.trac.wordpress.org/ticket/55985) seems to be coming for 6.2.

**Comment by justintadlock** [comment]
I've created a new Google Doc for March: https://docs.google.com/document/d/1l2CGDiXFRN6QhEzdiykGx779ozMehUM1ZnXD66ZbKd8/edit#

**Comment by abhansnuk** [comment]
Would this be better titled as “What’s new for WordPress developers? (March 2023)

**Reply by justintadlock** [reply]
Yep. The correct title is reflected in the [Google Doc](https://docs.google.com/document/d/1l2CGDiXFRN6QhEzdiykGx779ozMehUM1ZnXD66ZbKd8/edit#).

**Comment by bph** [comment]
This discussion is locked. Please add further comments to the issue 

- #84

**Comment by bph** [comment]
Published: [What’s new for developers? (March 2023)](https://developer.wordpress.org/news/2023/03/whats-new-for-developers-march-2023/)
